Libs/MRML/Core/vtkMRMLBitStreamVolumeNode.h

```diff
@@ -0,0 +1,110 @@
+#ifndef __vtkMRMLBitStreamVolumeNode_h
+#define __vtkMRMLBitStreamVolumeNode_h
+
+// MRML includes
+#include <vtkMRML.h>
+#include <vtkMRMLNode.h>
+#include <vtkMRMLStorageNode.h>
+#include "vtkMRMLVectorVolumeDisplayNode.h"
+#include "vtkMRMLVectorVolumeNode.h"
+#include "vtkMRMLVolumeArchetypeStorageNode.h"
+#include "vtkMRMLCompressionDeviceNode.h"
+
+// VTK includes
+#include <vtkStdString.h>
+#include <vtkImageData.h>
+#include <vtkObject.h>
+
+class  VTK_MRML_EXPORT vtkMRMLBitStreamVolumeNode : public vtkMRMLVectorVolumeNode
```

Comment:
Could you add a high-level description of the  class, its usage and relation ship to other class.